posts/crowdsales-and-multiple-inheritance.md

```diff
@@ -0,0 +1,327 @@
+On 2017 we saw a mind-blowing number of crowdsales. They have proved to be a
+powerful tool to collect the funds required to start a project, and they are
+one of the most common uses for smart contracts right now. The Zeppelin team
+has been very involved in this topic,
+[auditing many crowdsale contracts](https://blog.zeppelin.solutions/tagged/audit),
+and supporting [OpenZeppelin](https://openzeppelin.org/), the most popular
+framework to build these crowdsales. What we found was a lot of contracts
+repeating the same basic concepts, with very similar code, and common
+vulnerabilities.
+
+Earlier this year, part of the team took the task to redesign our base
+`Crowdsale` contract in order to support a lot of new crowdsale flavors out of
+the box and to make the experience of building and publishing your own
+crowdsale more clear and secure. The idea is to make audits less necessary,
```

Comment:
In the audits we observed that many were copy pasting the OZ code, with their own tweaks and customizations, and this was usually the place where we found vulnerabilities or mistakes.